src/sass/components/ConfirmationModal.scss

```diff
@@ -0,0 +1,35 @@
+.confirmation-modal, .confirmation-blurred-blocker {
+  position: absolute;
+  top:0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+}
+
+.confirmation-modal {
+  width: 30%;
+  min-width: 320px;
+  height: 200px;
+  margin: auto;
+  position: fixed;
+  z-index:3;
+  background:#e6e7e9;
```

Comment:
You can use something like `$color-gray-light` here instead (or `$color-gray-warm-light`).